### ISSUE TITLE
BatchNorm folding

### ISSUE THREAD

**Issue by ohjunee** [lutzroeder/netron#1178]
After exporting the convolution layers for the two cases below to onnx, I calculated the MACs and found that they had the same MACs value. I looked it up on [stackoverflow](https://stackoverflow.com/questions/69131168/why-is-it-that-when-viewing-the-architecture-in-netron-the-normalization-layer) found that BatchNorm folding is applied when exporting onnx, so I wonder if this is why the same MACs came out. If not, I think I should check if the mac calculation tool of the code I used is wrong.

```Conv3x3
Conv2dNormActivation(
  (0): Conv2d(64, 64, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
)

Conv3x3-bn
Sequential(
  (0): Conv2dNormActivation(
    (0): Conv2d(64, 64, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1), bias=False)
    (1): BatchNorm2d(64, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
  )
)

**Comment by lutzroeder** [comment]
@ohjunee please investigate if this is a visualization issue or a PyTorch question.

**Comment by ohjunee** [comment]
@lutzroeder The picture below shows netron's output for the two onnx models.

![image](https://github.com/lutzroeder/netron/assets/49416429/43030137-5acb-4b41-8e5d-22fa22ebc797)
